generics/services/sunbird.js

```diff
@@ -55,13 +53,12 @@ var organisationList = async function (token) {
   * @returns {JSON} - user profile information.
 */
 
-var getUserProfileInfo = function (token, userId) {
-    const createUserUrl =
-        process.env.sunbird_url + constants.endpoints.SUNBIRD_USER_READ + "/" + userId + "?fields=completeness,missingFields,lastLoginTime";
-
-
+const getUserProfileInfo = function (token, userId) {
     return new Promise(async (resolve, reject) => {
 
+        const createUserUrl =
+            process.env.SUNBIRD_URL + constants.endpoints.SUNBIRD_USER_READ + "/"
+            + userId + "?fields=completeness,missingFields,lastLoginTime";
```

Comment:
@rakeshSgr - Do we need these 3 fields. - fields=completeness,missingFields,lastLoginTime, how are we using it ?